### ISSUE TITLE
Params for loading pretrained basenet or checkpoints.

### ISSUE THREAD

**Issue by Lyns01** [dusty-nv/jetson-inference#1051]
What is the difference between the three parameters?
--base-net
--pretrained-ssd
--resume

**Comment by dusty-nv** [comment]
`--base-net` is the mobilenet backbone (i.e. no SSD in it yet)

`--pretrained-ssd` is a pretrained SSD model that was previously trained with train_ssd.py.   `--resume` is essentially the same minus some hyperparameter settings.  In most cases `--pretrained-ssd` seems to work better.

**Comment by Lyns01** [comment]
Thank you dusty.

**Comment by anish-faiz** [comment]
Good afternoon, I am training a custom object detection model using VGG16-SSD, instead of Mobilenet SSD for better accuracy. I have trained the model, but i could not load it. Can you please help me out to load my model and run detection.
